packages/cli/src/lib/bundler/server.ts

```diff
@@ -60,7 +60,17 @@ export async function serveBundle(options: ServeOptions) {
         // See https://github.com/facebookincubator/create-react-app/issues/387.
         disableDotRule: true,
       },
-      https: url.protocol === 'https:',
+      https:
+        url.protocol === 'https:'
+          ? {
+              cert: options.frontendConfig.getOptionalString(
```

Comment:
Think that you might want to use `backendConfig` here as the key will be secret and not available in `frontendConfig`.  A little misleading, but I think it's right.

Reply:
Hi @benjdlambert , thanks for all the comments! regarding this one, `backendConfig` does not seem to be present on the `options` object. the only was it seems to be able to inject the `key` variable here is from the bar `appConfig` object on the `options` (if it's going to remain `secret`) ?

Reply:
Hmm, ok I think what might be better is to return the `backendConfigs` in the `bundler/config.ts` instead and use those.

Something like this:

```ts
const frontendAppConfigs = schema.process(appConfigs, {
      visibility: options.fullVisibility
        ? ['frontend', 'backend', 'secret']
        : ['frontend'],
      withFilteredKeys: options.withFilteredKeys,
      withDeprecatedKeys: options.withDeprecatedKeys,
    });

    const backendAppConfigs = schema.process(appConfigs, {
      visibility: options.fullVisibility
        ? ['frontend', 'backend', 'secret']
        : ['backend'],
      withFilteredKeys: options.withFilteredKeys,
      withDeprecatedKeys: options.withDeprecatedKeys,
    });

    const frontendConfig = ConfigReader.fromConfigs(frontendAppConfigs);
    const backendConfig = ConfigReader.fromConfigs(backendAppConfigs);

    return {
      schema,
      appConfigs,
      frontendConfig,
      frontendAppConfigs,
      backendAppConfigs,
      backendConfig
    };
```

There's the frontend one already there, i just added it quickly in this snippet.